test/python/tests_to_run.txt

```diff
@@ -0,0 +1,8 @@
+math_ops_test.DivNoNanTest.testBasic
+#nn_ops
+math_ops_test.X*
+softmax_centered_test.S*
+softmax_op_test.Softmax#
```

Comment:
is this by design? that `#` anywhere is to be interpreted as comment (instead of just at the beginning)

Reply:
Yes, this is by design.  Anything after "#" needs to be ignored. Actually, I need to make few changes here for line 5 to work.

Reply:
anything after `#` is a comment. meaning `abc#def` means `abc`?
There are 3 options I suppose:
1. starts with # is a comment: `x[0]=='#'`
2. contains # is a comment. ignore full line: `'#' in x`
3. ignore after #, but take things before it.: `x.split('#')[0]`
option 3 is the most powerful one I guess

Reply:
Yes, that is right. I just implemented option #3